### PR TITLE
Remove quagga related code

### PR DESCRIPTION
**Pull request by shi-su**
#### Why I did it
Quagga is no longer being used. Remove quagga related code (e.g., docker-fpm-quagga, sonic-quagga, etc.). 

#### How I did it
Remove quagga related code.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: platform/p4/docker-sonic-p4.mk [hunk 1]
```diff
@@ -1,31 +0,0 @@
-# docker image for p4 sonic docker image
-
-DOCKER_SONIC_P4 = docker-sonic-p4.gz
```
Comment (qiluo-msft) [inline platform/p4/docker-sonic-p4.mk:3]: >p4 [](http://example.com/codeflow?start=31&length=2)

Is this related to quagga? Why remove?

Comment (shi-su) [reply platform/p4/docker-sonic-p4.mk:3]: Removing p4 related part is because it uses quagga regardless of the routing stack selected. Updated the PR description to clarify that.

File: rules/gobgp.mk [hunk 2]
```diff
@@ -1,5 +0,0 @@
-# gobgp package
```
Comment (qiluo-msft) [inline rules/gobgp.mk:1]: >gobgp [](http://example.com/codeflow?start=2&length=5)

gobgp itself is not related to quagga. Why remove?

Comment (shi-su) [reply rules/gobgp.mk]: Undid removal for gobgp.

Comment (lguohan) [reply rules/gobgp.mk]: let's keep gobgp and p4. only remove quagga.

**Review by qiluo-msft — APPROVED** [review]
LGTM. Please check with other reviewers.

**Comment by qiluo-msft** [comment]
@shi-su Could you update the PR title? I think it is not touching p4.
